src/functionBased/components/InputTodo.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-alert */
+/* eslint-disable react/prop-types */
```

Comment:
It would have been better if you gave your props a type rather than ignoring the linters.